src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -7,28 +7,43 @@
 import com.ctre.phoenix6.StatusSignal;
 import com.ctre.phoenix6.controls.DutyCycleOut;
 import com.ctre.phoenix6.controls.MotionMagicVelocityVoltage;
-import com.ctre.phoenix6.controls.PositionDutyCycle;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.hardware.CANcoder;
 import com.ctre.phoenix6.hardware.TalonFX;
 
 import frc.robot.constants.Hardware;
 import frc.robot.constants.shooter.ShooterConstants;
-
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class Shooter extends SubsystemBase
 {
   private final TalonFX m_ShooterMotor1 = new TalonFX(Hardware.SHOOTER_MOTOR_1_ID);
   private final TalonFX m_ShooterMotor2 = new TalonFX(Hardware.SHOOTER_MOTOR_2_ID);
   private final TalonFX m_WristMotor = new TalonFX(Hardware.WRIST_MOTOR_ID);
+  private final CANcoder m_WristCANCoder = new CANcoder(Hardware.SHOOTER_CANCODER_ID);
+
+  private final Timer m_timer;
 
-    /**
+  /**
    * @brief Creates a new Shooter subsystem.
   */
   public Shooter() 
   {
     m_ShooterMotor1.getConfigurator().apply(ShooterConstants.SHOOTER_1_GAINS);
     m_ShooterMotor2.getConfigurator().apply(ShooterConstants.SHOOTER_2_GAINS);
     m_WristMotor.getConfigurator().apply(ShooterConstants.WRIST_GAINS);
+
+    m_WristCANCoder.getConfigurator().apply(ShooterConstants.MAGNET_SENSOR_CONFIGS);
+
+    m_WristMotor.getConfigurator().apply(ShooterConstants.MOTION_MAGIC_CONFIGS);
```

Comment:
I have a question for this, instead of applying this configs multiple times, is it possible to keep them separately defined like how it is now, but at the end of the constants file combining them into a single config?

Reply:
Looking through the API, this is possible, I'll make the change